cpp/test/neighbors/ann_cagra.cuh

```diff
@@ -221,6 +245,10 @@ class AnnCagraTest : public ::testing::TestWithParam<AnnCagraInputs> {
  protected:
   void testCagra()
   {
+    // TODO (tarang-jain): remove when NN Descent index building support InnerProduct
```

Comment:
This is an impotant detail. Please create a Github issue for this and reference it in the code here so e don't lose sight of it.